src/main/java/org/folio/service/invoice/POLInvoiceLineRelationService.java

```diff
@@ -68,6 +69,9 @@ public Future<EncumbrancesProcessingHolder> manageInvoiceRelation(EncumbrancesPr
         if (forDelete.isEmpty()) {
           return Future.succeededFuture(encumbrancesProcessingHolder);
         }
+        if (forCreate.isEmpty()) {
```

Comment:
and what will be in this situation when forCreate is not empty and forDelete is not empty and contain different transasctions, in this case we should skip validation?
Maybe we should find transactions from forDetele that does not exist in forCreate and perform validation for them?

Reply:
please pay attention to the method copyAmountsAndRecalculateNewEncumbrance. In this method we will copy expended amount from encumbrances which intended for delition to encumbrances which will create